pkg/controller/podgc/gc_controller_test.go

```diff
@@ -672,15 +672,15 @@ func verifyDeletedAndPatchedPods(t *testing.T, client *fake.Clientset, wantDelet
 func testDeletingPodsMetrics(t *testing.T, inputDeletingPodsTotal int) {
 	t.Helper()
 
-	actualDeletingPodsTotal, err := metricstestutil.GetCounterMetricValue(deletingPodsTotal.WithLabelValues())
+	actualDeletingPodsTotal, err := metricstestutil.GetCounterMetricValue(deletingPodsTotal.WithLabelValues("out-of-service"))
 	if err != nil {
 		t.Errorf("Error getting actualDeletingPodsTotal")
 	}
 	if actualDeletingPodsTotal != float64(inputDeletingPodsTotal) {
 		t.Errorf("Expected desiredDeletingPodsTotal to be %d, got %v", inputDeletingPodsTotal, actualDeletingPodsTotal)
 	}
 
-	actualDeletingPodsErrorTotal, err := metricstestutil.GetCounterMetricValue(deletingPodsErrorTotal.WithLabelValues())
+	actualDeletingPodsErrorTotal, err := metricstestutil.GetCounterMetricValue(deletingPodsErrorTotal.WithLabelValues("out-of-service"))
 	if err != nil {
```

Comment:
Can you add unit tests for all the other labels?

Reply:
added.